www/config.yml

```diff
@@ -452,6 +452,19 @@ chapters:
   meetup_id: 248903240
   name: Ho Chi Minh, Vietnam
   twitter: pyladies1
+- email: hermosillo@pyladies.com
+  image: pyladies-hermosillo.jpeg
+  location:
+    latitude: 29.09
+    longitude: -110.95
+  meetup: PyLadiesHMO
+  name: Hermosillo, M&eacute;xico
+  organizer: Pyladies Hermosillo
+  website: hermosillo
```

Comment:
This is incorrect, please read comment in the pull request for how to remedy this.